qp_shotgun/humann2/humann2.py

```diff
@@ -89,8 +89,14 @@ def generate_humann2_analysis_commands(forward_seqs, reverse_seqs, map_file,
             % ', '.join(sn_by_rp.keys()))
 
     cmds = []
-    params = ['--%s "%s"' % (k, v) if v is not True else '--%s' % k
-              for k, v in viewitems(parameters) if v]
+    params = []
+    for k, v in viewitems(parameters):
+        if not v:
```

Comment:
Maybe use `if v is False or v == 'False'` instead, so that integer values of 0 don't accidentally skip?

```
In [2]: def paramtest(v):
   ...:     if not v:
   ...:         print('1')
   ...:     if v is True or v == 'True':
   ...:         print('2')
   ...:     else:
   ...:         print('3')
   ...:     return
   ...: 

In [3]: paramtest(0)
1
3
```